src/pages/EditOrganization.tsx

```diff
@@ -121,13 +123,19 @@ const UsersTable = () => {
     .catch(() => history.push('/dashboard'));
   }, []);
 
+  const onClickRemoveUser = (removedUser:OrganizationUser) =>
+  {
+    OrganizationService.deleteOrganizationUser(removedUser.organizationId,removedUser.userId);
```

Comment:
You still need to remove the user from the displayed list. For example:

```suggestion
    OrganizationService.deleteOrganizationUser(removedUser.organizationId, removedUser.userId).then(() => setOrgUsers(orgUsers.filter((orgUser) => orgUser.id !== removedUser.userId)));
```